README.md

```diff
@@ -32,12 +32,12 @@ Grants Content Kit Resources:
 - Melanie Calhoun
 - Melissa Barber
 - Sheila Powell
-- Tomer Tveria
 - Nicole Smith
 
 #### Summary & Accomplishments
 - [Created and committed sample content](https://docs.google.com/document/d/1gkLnFyskKXShowoOBk7ZaHHSDOMwTGbjIjhDpg-smWE/edit#heading=h.u35mx5t1qfmi)
 - [Investigated process to package Grant Content Kit](https://docs.google.com/document/d/1gkLnFyskKXShowoOBk7ZaHHSDOMwTGbjIjhDpg-smWE/edit#heading=h.693uk3rseyp6)
+- Updated GitHub Readme file so it's now more of a quick reference place, and moved documentation over to the Wiki in the repository.
 
 #### Recommendations for Future Sprints
-- 
+- Add demo video to the Readme file.
```

Comment:
```suggestion
- Add demo video to the Readme file.
- Edit and refine the help documentation in the Wiki.
- Move the help content in the Wiki into separate pages.
```

Reply:
OK, I added a few little bits about other documentation updates that still need to be done. @cidarm this update is ready to be updated and then this PR can be merged.